templates/shortcode-register.php

```diff
@@ -43,7 +43,7 @@
 			<input type="hidden" name="edd_honeypot" value="" />
 			<input type="hidden" name="edd_action" value="user_register" />
 			<input type="hidden" name="edd_redirect" value="<?php echo esc_url( $edd_register_redirect ); ?>"/>
-			<input class="button" name="edd_register_submit" type="submit" value="<?php esc_attr_e( 'Register', 'easy-digital-downloads' ); ?>" />
```

Comment:
The button class should be maintained for backwards compatibility.

Reply:
@pippinsplugins @amdrew Pairing `button` with `edd-submit` causes an issue again with the button colours:
![image](https://user-images.githubusercontent.com/1525554/29993599-a15c9108-8fb3-11e7-8bcd-4a1ac06a32a8.png)

Reply:
Ok that's fine then. Leave as is.